addon/templates/components/work-item-button.hbs

```diff
@@ -0,0 +1,19 @@
+<UkButton
+  ...attributes
+  @type={{@type}}
+  @disabled={{or @loading this.mutate.isRunning}}
```

Comment:
```suggestion
  @disabled={{or @disabled @loading this.mutate.isRunning}}
```

Reply:
You now changed it wrong for the second time. Just think a moment about what would make sense before you change it the next time :wink: